finagle-http/src/main/scala/com/twitter/finagle/http/Codec.scala

```diff
@@ -521,3 +522,95 @@ private[finagle] class HttpServerTracingFilter[Req <: Request, Res](serviceName:
     service(request)
   }
 }
+
+private[finagle] object OpenTracing {
+
+  import io.opentracing.Tracer
+  import io.opentracing.Span
+  import java.util.HashMap
+  import scala.collection.JavaConverters._
+
+  private[this] val REQUEST = "Finagle HTTP Request"
```

Comment:
would be better (if possible) to find out the finagle RPC method name (or whatever they call it) and use it here.